pkgs/build-support/fetchurl/mirrors.nix

```diff
@@ -99,6 +99,13 @@
     "http://www.ring.gr.jp/pub/net/"
   ];
 
+  # I2P
+  i2p = [
+    "https://download.i2p2.de/releases/"
+    "https://files.i2p-projekt.de/"
+    "https://download.i2p2.no/releases/"
+  ];
```

Comment:
If this is just used by one download we can also use urls there instead.

Reply:
Fixed, thanks!